src/main/java/io/confluent/examples/streams/MapFunctionLambdaExample.java

```diff
@@ -120,7 +121,7 @@ public static void main(final String[] args) throws Exception {
     final KStream<byte[], String> textLines = builder.stream("TextLinesTopic", Consumed.with(byteArraySerde, stringSerde));
 
     // Variant 1: using `mapValues`
-    final KStream<byte[], String> uppercasedWithMapValues = textLines.mapValues(String::toUpperCase);
+    final KStream<byte[], String> uppercasedWithMapValues = textLines.mapValues((ValueMapper<String, String>) String::toLowerCase);
```

Comment:
out of curiosity what is the issue?

Reply:
This:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.3:testCompile (default-testCompile) on project kafka-streams-examples: Compilation failure: Compilation failure:
[ERROR] /home/****/workspace/afka-streams-examples_PR-55-ZE4XP7RR5GNQKLRZHHKVNNWF2RAXJEI44CF2FVKHT2EHEL5ECZQA/src/test/java/io/confluent/examples/streams/FanoutLambdaIntegrationTest.java:[96,46] reference to mapValues is ambiguous
[ERROR] both method <VR>mapValues(org.apache.kafka.streams.kstream.ValueMapper<? super V,? extends VR>) in org.apache.kafka.streams.kstream.KStream and method <VR>mapValues(org.apache.kafka.streams.kstream.ValueMapperWithKey<? super K,? super V,? extends VR>) in org.apache.kafka.streams.kstream.KStream match
```

The added overload `mapValue(KeyValueMapper)` cannot be distinguished to the existing `mapValue(ValueMapper)` -- it's a lambda issue.

Reply:
oic. thanks